### PR TITLE
Added missing group vars.

### PR DESCRIPTION
**Pull request by ankanm**

### CODE REVIEW

File: ansible/group_vars/single [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+
+# Common variables for the 'single' group
```
Comment (arunabha) [inline ansible/group_vars/single]: More explanation of the single group or link to doc ?

File: ansible/group_vars/type_virtual [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+base_data_dir: /data
+
+tmp_dir: /tmp
+
+group_dirs_to_create: 
+  - /data
+  - /tmp
+
+san_mount: /san-mount
```
Comment (arunabha) [inline ansible/group_vars/type_virtual]: Do we expect all installs to have a SAN mount ?

File: ansible/group_vars/localmesos [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+# Common variables for local mesos cluster
+
+base_data_dir: /data
+tmp_dir: /tmp
+
+san_mount: /vagrant
```
Comment (arunabha) [inline ansible/group_vars/localmesos]: Why are we using SAN mount here ?

Comment (ankanm) [inline ansible/group_vars/localmesos]: It's used inside. We probably need to open an issue to fix/change this. But for now it's needed.

File: ansible/group_vars/single (hunk 1 above)
Comment (ankanm) [inline ansible/group_vars/single]: This one's used in vagrant single cluster.

File: ansible/group_vars/type_virtual (hunk 2 above)
Comment (ankanm) [inline ansible/group_vars/type_virtual]: Yes for now. But we should re-factor/update scripts to be more specific.

**Pull request closed without merging**